unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -717,15 +717,18 @@ protected void ConstructUnityTexture(TextureCreationSettings settings, byte[] bu
 
             //  NOTE: the second parameter of LoadImage() marks non-readable, but we can't mark it until after we call Apply()
             texture.LoadImage(buffer, false);
-
-            // We need to keep compressing in UNITY_EDITOR for the Asset Bundles Converter
-#if !UNITY_STANDALONE || UNITY_EDITOR
-            if ( Application.isPlaying )
+            
+            if (Application.isPlaying)
             {
+                
+                // We need to keep compressing in UNITY_EDITOR for the Asset Bundles Converter
+#if UNITY_EDITOR
```

Comment:
Could this be a "IS_RUNNING_AB" instead of "UNITY_EDITOR"? It may be better to check that feature flags are working

Reply:
In the end I discovered that we were already avoiding texture compressions with the Asset Bundle converter so I removed any comment regarding that.